packages/sanity/src/core/releases/store/useDocumentVersionInfo.ts

```diff
@@ -15,6 +15,8 @@ function exists(value: any) {
 }
 const DOCUMENT_STUB_PATHS = ['_id', '_type', '_rev', '_createdAt', '_updatedAt']
 
+const NO_VERSIONS = {} as Record<string, VersionInfoDocumentStub | undefined>
```

Comment:
You sholud use `Object.freeze({} as Record<string, VersionInfoDocumentStub | undefined>)` here to prevent another issue where someone modifies this object

Reply:
It would probably also be help to mark it as `ReadOnly<>`, which is what freeze does in TypeScript.

Reply:
Oh, yeah, that's a great idea!